services/api/mw2pdf/index.js

```diff
@@ -30,10 +32,20 @@ program
     // Process passthrough parameters
     let processedUrls = urls
     if (options.passthroughParameters) {
-      const passthroughParameters = JSON.parse(options.passthroughParameters)
-      processedUrls = processedUrls.map(
-        (url) => updateUrlParameters(url, passthroughParameters),
-      )
+      try {
+        assertValidPassthroughParameters(options.passthroughParameters)
+        const passthroughParameters = JSON.parse(options.passthroughParameters)
+        processedUrls = processedUrls.map(
+          (url) => updateUrlParameters(url, passthroughParameters),
+        )
+      } catch (e) {
+        if (e instanceof Joi.ValidationError) {
```

Comment:
I wish that JS had conditional catch clauses :-D

Reply:
Agreed :(

[This does exist but I decided not to even try it](https://stackoverflow.com/questions/20483597/conditional-catch-clauses-browsers-support/42607272)